### PR TITLE
Update backing up and restoring page of the accounts section for the …

### PR DESCRIPTION
**Pull request by emresurmeli**
1. Link mnemonic phrase
2. Update images to recent version of the polkadot.js UI
3. Add new section "Transferring Polkadot-JS Apps Accounts/Addresses From One Computer to Another"

### CODE REVIEW

File: docs/learn/learn-account-restore.md [hunk 1]
```diff
@@ -78,7 +78,14 @@ creating a new account, rather adding that account onto the Polkadot-JS UI. Any
 same seed words will have control over that account on-chain. This is why it is so important to keep
 your seed words secret and safe.
 
-![restore-using-json](../assets/accounts/polkadot-js-existing-json.png)
+Step 1
+![restore-using-json](../assets/accounts/backing_up_and_restoring/via_existing_seed_1.png)
```
Comment (profradha) [inline docs/learn/learn-account-restore.md]: The alt text for the three newly added images should provide context for the image. Right now it is restore-using-json for all the three image files

File: docs/learn/learn-account-restore.md [hunk 2]
```diff
@@ -105,3 +112,41 @@ Set an identity PIN that will be used to unlock this account when you need to.
...
+
+1. Go to Polkadot-JS Apps
+2. Go to JavaScript console (on Brave, NOT on Polkadot-JS)
+3. Type in the command JSON.stringify(localStorage)
+4. Copy and paste the returned string to a text editor.
+5. Check that the string you pasted both begins and ends with a tick mark ('). If not, add one to the beginning and end.
+6. Save and send that JSON file to the new computer.
+7. On new computer, go to Polkadot-JS Apps
+8. Open the Javascript console
+9. Set a variable raw equal to the JSON you created =
```
Comment (profradha) [inline docs/learn/learn-account-restore.md]:
```suggestion
9. Set a variable raw equal to the string from the text file
```

File: docs/learn/learn-account-restore.md [hunk 3]
```diff
@@ -105,3 +112,41 @@ Set an identity PIN that will be used to unlock this account when you need to.
...
+> This generally should not make a difference (since it can still access the same account), but might if you 
+> have e.g. an account which was stored externally in the extension on the old computer but was created 
+> directly in browser on the new one.
+
+This has been tested on Brave and Chrome, but not other browsers.
+
+1. Go to Polkadot-JS Apps
+2. Go to JavaScript console (on Brave, NOT on Polkadot-JS)
+3. Type in the command JSON.stringify(localStorage)
+4. Copy and paste the returned string to a text editor.
```
Comment (profradha) [inline docs/learn/learn-account-restore.md]:
```suggestion
4. Copy and paste the returned string to a text editor and save the file.
```

File: docs/learn/learn-account-restore.md [hunk 4]
```diff
@@ -105,3 +112,41 @@ Set an identity PIN that will be used to unlock this account when you need to.
...
+5. Check that the string you pasted both begins and ends with a tick mark ('). If not, add one to the beginning and end.
+6. Save and send that JSON file to the new computer.
```
Comment (profradha) [inline docs/learn/learn-account-restore.md]:
```suggestion
6. Save and send that file with the copied string to the new computer.
```

File: docs/learn/learn-account-restore.md [hunk 5]
```diff
@@ -105,3 +112,41 @@ Set an identity PIN that will be used to unlock this account when you need to.
...
+7. On new computer, go to Polkadot-JS Apps
+8. Open the Javascript console
```
Comment (profradha) [inline docs/learn/learn-account-restore.md]:
```suggestion
8. Open the Javascript console on the browser (Available in Developer Tools)
```

File: docs/learn/learn-account-restore.md [hunk 6]
```diff
@@ -105,3 +112,41 @@ Set an identity PIN that will be used to unlock this account when you need to.
...
+
+> **Note:** Note: This will overwrite any existing accounts with the same pubkey on your new computer. 
+> This generally should not make a difference (since it can still access the same account), but might if you 
+> have e.g. an account which was stored externally in the extension on the old computer but was created 
+> directly in browser on the new one.
+
+This has been tested on Brave and Chrome, but not other browsers.
+
+1. Go to Polkadot-JS Apps
+2. Go to JavaScript console (on Brave, NOT on Polkadot-JS)
```
Comment (profradha) [inline docs/learn/learn-account-restore.md]:
```suggestion
2. Go to JavaScript console on the browser (Available in Developer Tools)
```

File: docs/learn/learn-account-restore.md [hunk 7]
```diff
@@ -105,3 +112,41 @@ Set an identity PIN that will be used to unlock this account when you need to.
...
+3. Type in the command JSON.stringify(localStorage)
```
Comment (profradha) [inline docs/learn/learn-account-restore.md]: Just the way code is shown distinct from the text in the steps below, ensure this line below is displayed as code

JSON.stringify(localStorage)

File: docs/learn/learn-account-restore.md [hunk 8]
```diff
@@ -105,3 +112,41 @@ Set an identity PIN that will be used to unlock this account when you need to.
...
+6. Save and send that JSON file to the new computer.
+7. On new computer, go to Polkadot-JS Apps
+8. Open the Javascript console
+9. Set a variable raw equal to the JSON you created =
+
+```
+raw = ... copy-pasted json from original computer ...
+```
+
+10. Run the following code:
```
Comment (profradha) [inline docs/learn/learn-account-restore.md]:
```suggestion
10. Run the following code on the console:
```

**Review by DrW3RK — CHANGES_REQUESTED** [review]
Minor updates to the tutorial instructions (I ran through all the steps and they work)

Also, @emresurmeli - As we recommend usage of Polkadot-JS Extension, it makes sense to update the tutorial with instructions on how to restore the account using mnemonic phrase on the extension rather than on the Polkadot JS Apps UI. It is okay to retain the existing instructions stating that it is always recommended to use the extension instead of the Apps UI

**Review by DrW3RK — APPROVED** [review]

**Pull request merged**